.changeset/small-months-heal.md

```diff
@@ -5,6 +5,6 @@
 "section": tree
 ---
 
-Branch merges generate revertibles
+Branch merges now generate revertibles
```

Comment:
@noencke Is this referring to the alpha branching APIs that were added in this same release? I'm wondering if we should collapse this information into the announcement of the new APIs rather than a separate item. What do you think?

Reply:
Good question. Yes - merges cannot be accomplished without the alpha branching API, so this is only relevant for that API. I think you're right that they would be best combined.

Reply:
Nit: I'd update line 10 to read "via the `TreeBranch.merge` method".